frontend/components/feedback-task/DatasetFilters.component.vue

```diff
@@ -52,7 +62,37 @@ export default {
         order: 0,
         placeholder: "Introduce your query",
       },
+      statusSelector: {
+        id: "statusSelector",
+        name: "Status Selector",
+        componentType: "statusSelector",
+        order: 1,
+        options: [
+          {
+            id: "pending",
+            name: "Pending",
+            color: "#B6B9FF",
+            textColor: "#4C4EA3",
+            bgColor: "#EEEEFF",
+          },
+          {
+            id: "submitted",
+            name: "Submitted",
+            color: "#3E5CC9",
+            textColor: "#3E5CC9",
+            bgColor: "#EBF3FF",
+          },
+          {
+            id: "discarded",
+            name: "Discarded",
+            color: "#B7B7B7",
+            textColor: "#656363",
+            bgColor: "#F2F2F2",
```

Comment:
the color properties bgColor/textColor need to be manage in the stupid component